static/app/components/forms/controls/rangeSlider/slider.tsx

```diff
@@ -0,0 +1,158 @@
+import styled from '@emotion/styled';
+
+export default styled('input')<{hasLabel: boolean}>`
```

Comment:
Not to start this conversation again, but I actually thought we had a lint rule that required default exports to have names?

Reply:
https://github.com/getsentry/eslint-config-sentry/pull/139 here's the fix @vbro go ahead and give these names like

```tsx
const Slider = styled('input')<{hasLabel: boolean}>`
  ...
`;

export default Slider;
```